README.md

```diff
@@ -545,36 +545,38 @@ TrackScreenTimeRequest.Builder()
 
 ##### Track Purchase Request Object
 
-| Parameter              | Type                    | Field      |
-| ---------------------- | ----------------------- | ---------- |
-| skuName                | String                  | Required   |
-| skuDescription         | String                  | Optional   |
-| skuType                | int                     | Required   |
-| skuSubType             | int                     | Required   |
-| purchaseType           | int                     | Required   |
-| purchaseCurrencyType   | int                     | Required   |
-| purchasePrice          | float                   | Required   |
-| percentDiscount        | int                     | Optional   |
-| amountRewarded         | int                     | Optional   |
-| skuName                | String                  | Optional   |
-| skuDescription         | String                  | Optional   |
-| skuType                | String                  | Required   |
-| skuSubType             | String                  | Required   |
-| timestamp              | String                  | Required   |
-
-```
+| Parameter               | Type                                         | Field      |
+| ----------------------  | -------------------------------------------- | ---------- |
+| skuName                 | String                                       | Required   |
+| skuDescription          | String                                       | Optional   |
+| primaryPurchaseCategory | PurchaseCategory                             | Required   |
+| purchaseType            | PurchaseType                                 | Required   |
+| purchaseCurrencyType    | PurchaseCurrencyType                         | Required   |
+| purchasePrice           | float                                        | Required   |
+| percentDiscount         | int                                          | Optional   |
+| amountRewarded          | int                                          | Optional   |
+| skuName                 | String                                       | Optional   |
+| skuDescription          | String                                       | Optional   |
+| skuType                 | String                                       | Required   |
+| skuSubType              | String                                       | Required   |
+| purchaseDetails         | List<TrackPurchaseRequest.PurchaseDetails>   | Required   |
+| timestamp               | String                                       | Required   |
+
+```
+PurchaseCategory primaryPurchaseCategory = PurchaseCategory.Currency.GOLD;
```

Comment:
@Debadri-Redapple This works pretty well. I have a small proposal.

PurchaseDetail -> `PurchaseDetails`

```
TrackPurchaseRequest.Builder()
    .SkuName(skuName)
    .SkuDescription(skuDescription)
    .PurchaseCategory(purchaseCategory)
    .PurchaseType(purchaseType)
    .PurchaseCurrencyType(purchaseCurrencyType)
    .PurchasePrice(purchasePrice)
    .PercentDiscount(percentDiscount)
    .AmountRewarded(amountRewarded)
    .PurchaseDetail(purchaseDetails) // <-- PurchaseDetails(purchaseDetails)
    .Create(
        trackAppOpenRequest => MikrosManager.Instance.AnalyticsController.LogEvent(trackAppOpenRequest),
        onFailure =>
        {
            Debug.Log("Unrecognized Error Occured");
        }
	);
```

Also, I am confused about this part (below). 
-- Why the trackAppOpenRequest? Do you mean to say something else here such as trackPurchaseRequest?

```
.Create(
        trackAppOpenRequest => MikrosManager.Instance.AnalyticsController.LogEvent(trackAppOpenRequest),
        onFailure =>
        {
            Debug.Log("Unrecognized Error Occured");
        });
```